.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -45,6 +45,7 @@ assignees: ''
 - [ ] Bump PyGMT version on https://github.com/GenericMappingTools/try-gmt (after conda-forge update)
 - [ ] Announce the release on:
   - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/) (do this announcement first! draft on https://hackmd.io/@pygmt. requires moderator status)
+  - [ ] [ResearchGate](https://www.researchgate.net) (after forum announcement, add new version as research item via the *code* category, be sure to include the corresponding new ZENODO DOI)
```

Comment:
This was fast! Thanks @michaelgrund.

So far, we do not capitalize "Zenodo":
```
$ grep "Zenodo" *
AUTHORSHIP.md:## Authorship on Zenodo archives of releases
AUTHORSHIP.md:be invited to be an author on the Zenodo archive of new releases.
AUTHORSHIP.md:If you have contributed and do not wish to be included in Zenodo archives,
AUTHORSHIP.md:   included with something like `(not included in Zenodo)`.
AUTHORSHIP.md:Note that authors included in the Zenodo archive will also have their name
grep: ci: Is a directory
grep: doc: Is a directory
grep: examples: Is a directory
grep: pygmt: Is a directory
README.rst:    :alt: Digital Object Identifier for the Zenodo archive
README.rst:      publisher    = {Zenodo},
README.rst:To cite a specific version of PyGMT, go to our Zenodo page at
```

```suggestion
  - [ ] [ResearchGate](https://www.researchgate.net) (after forum announcement, add new version as research item via the *code* category, be sure to include the corresponding new Zenodo DOI)
```
-----
And one minor thought: In the related issue comment, you highlighted "code" in bold, but here it is italic 🙂. I have no strong preference regarding one of these styles.

Reply:
I agree with @yvonnefroehlich's two points. Otherwise, this PR looks good to me.